### PR TITLE
specify types for defaultProps in mesh layer

### PR DESCRIPTION
**Pull request by XiaokaiUber**
For #2477 

#### Background
In mesh layer, the defaultProps don't have types specified. We need to specify type for those props.

#### Change List
- Add type to the defaultProps in the mesh layer

note: tested with render test and layer browser

### CODE REVIEW

File: modules/experimental-layers/src/mesh-layer/mesh-layer.js [hunk 1]
```diff
@@ -105,14 +105,14 @@ const defaultProps = {
   // Optional settings for 'lighting' shader module
   lightSettings: {},
 
-  getPosition: x => x.position,
-  getColor: x => x.color || DEFAULT_COLOR,
+  getPosition: {type: 'accessor', value: x => x.position},
+  getColor: {type: 'accessor', DEFAULT_COLOR},
```
Comment (1chandu) [inline modules/experimental-layers/src/mesh-layer/mesh-layer.js]: don't we need `value: x => x.color`, same for the ones below.

File: modules/experimental-layers/src/mesh-layer/mesh-layer.js [hunk 2]
```diff
@@ -137,9 +136,13 @@ export default class MeshLayer extends Layer {
       instanceRotations: {
         size: 3,
         accessor: ['getYaw', 'getPitch', 'getRoll'],
-        update: this.calculateInstanceRotations
+        defaultValue: [0, 0, 0]
```
Comment (1chandu) [inline modules/experimental-layers/src/mesh-layer/mesh-layer.js]: should we match the order of array elements in `accessor` with how they used in `calculateInstanceRotations` ?

**Comment by Pessimistress** [comment]
> Add a render test for mesh-layer

There is already a render test for path marker layer, which uses mesh layer. Render test does not pass on this branch, please fix it.

There is also a mesh layer test in layer browser, where you can test a combination of prop values.

File: modules/experimental-layers/src/mesh-layer/mesh-layer.md [hunk 3]
```diff
@@ -99,7 +99,7 @@ See [Euler angles](https://en.wikipedia.org/wiki/Euler_angles).
 
 ##### `getColor` (Function, optional)
 
-- Default: `object => object.color || [0, 0, 0, 255]`
+- Default: `object => [0, 0, 0, 255]`
```
Comment (Pessimistress) [inline modules/experimental-layers/src/mesh-layer/mesh-layer.md]: Default is not a function.

**Pull request merged**